src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig

```diff
@@ -101,6 +101,12 @@
   {{ block('form_help') }}
 {% endblock switch_widget %}
 
+{%- block url_widget -%}
```

Comment:
This modification is in `prestashop_ui_kit.html.twig` and not `prestashop_ui_kit_base.html.twig` is that intended ?

Reply:
Good catch, yeah I think it should be in base.